learners/reference.md

```diff
@@ -97,6 +97,6 @@ Git cheat sheet handouts:
 
 - The [help pages](https://help.github.com/) of GitHub are a good place to start
 - GitHub has '[activities](https://guides.github.com/activities/hello-world/)' which aim to explain how git works
-- GitHub also has interactive tutorials for their [online version (Learning Labs)](https://lab.github.com/) and for [using Git offline (Git-It)](https://github.com/jlord/git-it-electron#git-it-desktop-app)
+- GitHub also has interactive tutorials for their [online version (GitHub Skills)](https://skills.github.com/) and for [using Git offline (Git-It)](https://github.com/jlord/git-it-electron#git-it-desktop-app)
```

Comment:
This is great, but it's no longer needed as this change was merged from pull request #183 (submitted a matter of seconds before this one, it seems).